# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: docshell/test/browser/file_bug1415918_beforeunload_iframe.html
# Commit: 4b76024db21c
# Full Hash: 4b76024db21cff006a7424a35c412c6e0a153eb0
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2020-09-22 03:47:42
# Regressor Bug: 1655866
# File Overlap Count: 1
# Description:
#   Bug 1655866: Part 5 - Use native PermitUnload implementation from front-end code. r=nika,mconley
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D88318
# ==============================================================================

diff -r 55fb069319f1 -r 4b76024db21c docshell/test/browser/file_bug1415918_beforeunload_iframe.html
--- a/docshell/test/browser/file_bug1415918_beforeunload_iframe.html	Mon Sep 21 22:41:01 2020 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,18 +0,0 @@
-<html>
-<head>
-  <style>
-    body[fired] {
-      background-color: #0f0;
-    }
-  </style>
-  <script>
-    window.onbeforeunload = function() {
-      document.body.setAttribute("fired", true);
-      return "Frame document prompt";
-    };
-  </script>
-</head>
-</body>
-FRAME
-</body>
-</html>